### PR TITLE
[FEATURE][MS-94] Homepage - List all Job Opportunities

### PR DESCRIPTION
**Pull request by juanandres7698**

### CODE REVIEW

File: src/Components/Home/home.module.css [hunk 1]
```diff
@@ -1,6 +1,100 @@
 .container {
   flex: 1;
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
-}
\ No newline at end of file
+  width: 100%;
+  margin: 0 auto;
```
Comment (BaltazarCortina) [inline src/Components/Home/home.module.css:8]: Add `max-width: 1200px` here

File: src/Components/Home/home.module.css [hunk 2]
```diff
@@ -1,6 +1,100 @@
...
+}
+.boxesContainer {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  width: 100%;
+}
+.boxContainer {
+  display: flex;
+  width: 30%;
```
Comment (BaltazarCortina) [inline src/Components/Home/home.module.css:61]: Add `padding: 10px` here

File: src/redux/home/reducer.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  isFetching: false,
+  list: [],
+  selectedElement: {
+    idCompany: '',
+    idProfile: '',
+    startDate: '',
+    endDate: '',
+    jobDescription: ''
+  },
+  options: { companies: [], profiles: [] },
```
Comment (BaltazarCortina) [inline src/redux/home/reducer.js:18]: Remove these lines.

File: src/redux/home/reducer.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+};
+
+const homePositionsReducer = (state = initialState, action) => {
+  switch (action.type) {
+    // GET POSITIONS
+    case GET_HOMEPOSITIONS_FETCHING:
+      return {
+        ...state,
+        isFetching: true,
+        selectedElement: {},
```
Comment (BaltazarCortina) [inline src/redux/home/reducer.js:29]: Remove this line.

File: src/redux/home/reducer.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      return {
+        ...state,
+        isFetching: false,
+        selectedElement: {
+          idCompany: '',
+          idProfile: '',
+          startDate: '',
+          endDate: '',
+          jobDescription: ''
+        },
```
Comment (BaltazarCortina) [inline src/redux/home/reducer.js:56]: Remove these lines.

File: src/redux/home/thunks.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+} from './actions';
+
+export const getHomePositions = () => {
+  return (dispatch) => {
+    dispatch(getHomePositionsFetching());
+    fetch(`${url}/open-positions`, {
+      headers: {
+        'Content-Type': 'application/json',
+        token: sessionStorage.getItem('token')
+      }
```
Comment (BaltazarCortina) [inline src/redux/home/thunks.js:16]: This route is public, token is not needed

**Review by BaltazarCortina — APPROVED** [review]

**Review by Sabrucita — APPROVED** [review]

**Pull request merged**